docs: PPE+U pricing model for creators (#2235)
apify/apify-docs: sources/platform/actors/publishing/monetize/pay_per_event.mdx

1. _Test your pricing_: Run your Actor and analyze cost-effectiveness using a special dataset.
1. _Communicate value_: Ensure pricing reflects the value provided and is competitive.

## Pass platform usage to users

> If you have difficulty estimating event prices and want to avoid negative profit margins, pass the platform usage costs directly to your users. With this option enabled, users pay for both the defined events and the underlying platform usage.

Reviewer: ```suggestion
If you cannot accurately estimate event prices upfront and want to avoid negative profit margins, you can pass the platform usage costs directly to users. Users then pay both the defined events and the underlying platform usage.
```

:::caution Less transparent pricing

> This pricing model is less transparent for users. Disable this option once you determine accurate event pricing.

Reviewer: ```suggestion
Charging platform usage separately is less transparent for users. Turn off this option once you determine accurate event pricing.
```

> Charging platform usage separately is less transparent for users. Turn off this option once you determine accurate event pricing as it negatively impacts your **Actor quality score**.

Reviewer 2: It seems a bit too mild. User should not do this for "Actor Q score" but mainly for the pricing transparency. 
Maybe something like
```
Charging platform usage separately is less transparent for users and may discourage them from using your Actor.  This also negatively impacts your [**Actor quality score**](I would add a link here, not everyone know what it is). Turn off this option once you determine accurate event pricing.
```

Author: I really like this suggestion. I felt like something was missing, too. - `may discourage them from using your Actor` this is it.

:::

Turn on the **Pay per event + usage** toggle to enable this option.

![Pay per event with usage toggle button](../images/pay_per_event_and_usage_toggle.png)

> To stop passing platform usage to users, update your pricing model and turn off the **Pay per event + usage** toggle.

Author: The pricing model should have referred to the PPE+U because it charges users differently than regular PPE but maybe it's unnecessarily adding a new term which only brings confusion.
